src/main/resources/kami_at.cfg

```diff
@@ -65,8 +65,14 @@ public net.minecraft.tileentity.TileEntity field_145854_h # blockType
 public net.minecraft.item.ItemTool field_77865_bY # attackDamage
 public net.minecraft.util.Timer * # All fields
 public net.minecraft.client.shader.ShaderGroup field_148031_d # listShaders
+public net.minecraft.client.gui.MapItemRenderer * # All fields
```

Comment:
All methods should include all fields

Reply:
I don't think it does. It might be the other way around though if that is what you mean.

Reply:
Uhhhhhhhh ngl I'm not entirely sure. I could be wrong here, I'll confirm when I can